operator/sp1/lib/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "sp1-verifier-ffi"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+bincode = "1.3.3"
+# sp1-core = { git = "https://github.com/succinctlabs/sp1.git" }
+# sp1-sdk = { git = “https://github.com/succinctlabs/sp1.git”, rev = “v1.0.0-testnet” }
```

Comment:
just a reminder for removing this comments

Reply:
Removed on 4c719e0e9772da33371d3692160444d313adb38f